src/gradient/sensitivity_calculator.py

```diff
@@ -38,28 +58,25 @@ def __init__(self, data: DataLoader, model: str):
 
     def _select_ngm_calculator(self):
         """
-        Select the appropriate NGMCalculator based on the model name
+        Select the appropriate NGMCalculator based on the model name.
         """
-        if self.model == "chikina":
-            self.ngm_calculator_class = ChikinaNGMCalculator
-        elif self.model == "moghadas":
-            self.ngm_calculator_class = MoghadasNGMCalculator
-        elif self.model == "seir":
-            self.ngm_calculator_class = SeirNGMCalculator
-        elif self.model == "rost":
-            self.ngm_calculator_class = RostNGMCalculator
-        elif self.model == "italy":
-            self.ngm_calculator_class = ItalyNGMCalculator
-        elif self.model == "kenya":
-            self.ngm_calculator_class = KenyaNGMCalculator
-        elif self.model == "british_columbia":
-            self.ngm_calculator_class = BCNGMCalculator
-        else:
+        model_map = {
```

Comment:
This variable can go to `src.models.__init__.py` and you can refer to it as `models.model_map`. Since this is a dictionary, you can simply refer to an element with `models.model_map[self.model]`.